ignite/contrib/handlers/lr_finder.py

```diff
@@ -264,18 +263,17 @@ def plot(
             losses = losses[skip_start:-skip_end]
 
         # Plot loss as a function of the learning rate
-        plt.plot(lrs, losses)
+        if axes is None:
+            f, axes = plt.subplots(**kwargs)
```

Comment:
This part of code should be executed before `if display_suggestion:` and we have to remove `fig, ax = plt.subplots()`, line 252.